### DISCUSSION TITLE
Coosa Plug lost blue LED for connection

### DISCUSSION THREAD

**Discussion by Siggi0904** [arendst/Tasmota#21246] [Q&A]
Hi,

my Coosa Plugs https://templates.blakadder.com/coosa.html get lost the blue connection LED with latest stable 13.4.0

I cannot switch these second LED on.
The red on/off LED work well.

Thanks for feedback and possible fix.

**Comment by s-hadinger** [comment]
I cannot reproduce. I configure a similar configuration on Wemos D1 Mini, with Relay + Led1 + Led2. Led1 is displaying the connection status (blinking until it's connected), and Led2 mimics the state of the relay.

I took the stock 13.4.0 release.

**Comment by Siggi0904** [comment]
Hm, all my blue leds of my sockets are off.
Only red ones goes on/off if I switch the socket.

Before 13.4 all was fine.

How I can check anything or can I get the blue wifi LED back to work?

**Reply by s-hadinger** [reply]
The only way would be to reproduce on a general Dev board with similar CPU. As said, I reproduced a similar template on ESP8266 and LED2 works fine.

Now I have no idea which one is red and which one is blue, nor what blue led was supposed to show
